internal/local/app.go

```diff
@@ -251,6 +266,32 @@ func (a *App) LoadConfig() error {
 	return nil
 }
 
+// LoadAppMeta will load the local app's data from .mdb/app_meta.json
+func (a *App) LoadAppMeta() error {
+	path := filepath.Join(a.RootDir, FileMdbAppMeta.String())
+	data, dataErr := ioutil.ReadFile(path)
+	if dataErr != nil {
+		return errFailedToParseAppConfig(path)
+	}
+
+	if err := json.Unmarshal(data, &a.AppMeta); err != nil {
+		return errFailedToParseAppConfig(path)
+	}
+
+	switch a.AppMeta.ConfigVersion {
```

Comment:
not sure how this switch statement is beneficial here. 
The config is being set right before calling `LoadConfig`

```
app := App{RootDir: path, Config: config}
if err := app.LoadConfig(); err != nil {
 	return App{}, false, err
}
```

Reply:
I would actually prefer to see us avoid this here as well... `LoadAppMeta` should do just that

the logic inside `FindApp`, which should attempt to `LoadAppMeta` first before doing our iterative check of known root config files is where we'll be able to manage this stuff